beaker.gemspec

```diff
@@ -43,6 +42,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'open_uri_redirections', '~> 0.2.1'
   s.add_runtime_dependency 'in-parallel', '~> 0.1'
   s.add_runtime_dependency 'thor', '~> 0.19'
+  s.add_runtime_dependency 'pry-byebug'
```

Comment:
https://github.com/puppetlabs/beaker/pull/1436 conflicts/supersedes with this change and it does specify a version number

I think I'd prefer for https://github.com/puppetlabs/beaker/pull/1436 to be merged and then this PR to just become a docs change